lnbits/db.py

```diff
@@ -29,6 +29,13 @@ def interval_seconds(self, seconds: int) -> str:
             return f"{seconds}"
         return "<nothing>"
 
+    def datetime_to_timestamp(self, date: datetime.datetime):
+        if self.type in {POSTGRES, COCKROACH}:
+            return date.strftime("%Y-%m-%d %H:%M:%S")
+        elif self.type == SQLITE:
+            return time.mktime(date.timetuple())
+        return "<nothing>"
```

Comment:
Helper function that converts `datetime.datetime` objects to a suitable format for database storage. Uses `Timestamp` for Postgres and Unix timestamps for SQLite.